channeldb/migration_12_mc_final_cltv.go

```diff
@@ -0,0 +1,20 @@
+package channeldb
+
+import "github.com/coreos/bbolt"
+
+// migrateMcFinalCltv clears mission control to start with a clean slate on
+// which final cltv delta values are stored too. Because mission control data is
+// not critical we opt for simply clearing the existing data rather than a
+// read/update/write migration.
+func migrateMcFinalCltv(tx *bbolt.Tx) error {
```

Comment:
Shall we commence the new migration package proposal with this PR? As it's the first target migration for 0.9,  and the migration itself is pretty light. 

More on topic for this PR: perhaps we should just start migrating this data? It's not critical, but certain services could see degradation in service quality after updating beyond this PR. We currently don't have a way of _loading_ external data, so there's no way for them to preserve the mission control data if they consider it precious enough to do so.